go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/savsgio/atreugo/v10
 go 1.13
 
 require (
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
Remove this line, it's a dependency of jwt example, please.

Reply:
OK, I removed. The problem is that since it's under examples every time I run `go build ./...` or ` golangci-lint` it gets added back